web/src/tippyjs.js

```diff
@@ -262,25 +300,16 @@ export function initialize() {
     delegate("body", {
         target: ".recipient_bar_icon",
         onShow(instance) {
-            if (!document.body.contains(instance.reference)) {
-                return false;
+            // Don't push the instance to the tippyInstances array if it already exists.
+            if (!tippyInstances.includes(instance)) {
+                tippyInstances.push(instance);
             }
```

Comment:
Since `onCreate` is called before `onShow`, I don't think it is required.

Reply:
I included it to handle any edge cases where we might miss catching an instance and to avoid duplicates also added a condition, but your reasoning is correct, and I have now removed it